src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public class Lottery {
+    public String getRandomBall(Ball ball) {
+        return " color - " + ball.getColor() + " , number - " + ball.getNumber();
```

Comment:
You must use CollorSupplier and Random here to create an object Ball class with random fields